test/fixture/nitro.config.ts

```diff
@@ -1,8 +1,9 @@
+import { defineConfig } from "nitro";
+
 import { fileURLToPath } from "node:url";
-import { defineNitroConfig } from "nitro/config";
 import { dirname, resolve } from "node:path";
 
-export default defineNitroConfig({
```

Comment:
Actually, this name was good for distinguishing.

Reply:
You can do `{ defineConfig as defineNitroConfig }`. Shorter is simpler and similar to vite's `defineConfig`